src/gdata.h

```diff
@@ -29,40 +29,34 @@
 
 ******************************************************************************/
 
-#define KPP_VERSION "2.2.3"
+#define KPP_VERSION "3.0.0"
 
 #ifndef _GDATA_H_
 #define _GDATA_H_
 
 #include <stdio.h>
 
-/* - Many limits can be changed here by adjusting the MAX_* constants */
-/* - To increase the max size of inlined code (F90_GLOBAL etc.), */
-/*   change MAX_INLINE in scan.h */
-
-// Note: MAX_EQN or MAX_SPECIES over 1023 causes a seg fault in CI build -- lae 10/13/2021
-#define MAX_EQN       11000
-#define MAX_SPECIES    6000
-#define MAX_SPNAME       30
-#define MAX_IVAL         40
-/* MAX_EQNTAG = max length of equation tag in eqn file */
-/* Check if changes in code_f90.c are also necessary when changing MAX_EQNTAG */
-#define MAX_EQNTAG       32
-/* To allow longer f90 expressions for the rate coefficients in the */
-/* eqn file, change MAX_K here and union str in scan.y consistently */
-#define MAX_K          1000
-#define MAX_ATOMS	 10
-#define MAX_ATNAME	 10
-#define MAX_ATNR	250 
-#define MAX_PATH        250
-#define MAX_FILES	 20
-#define MAX_FAMILIES	 50
-#define MAX_MEMBERS 	150
-/* MAX_EQNLEN only determines how long the printout of the */
-/* equation in the Monitor file will be. */
-#define MAX_EQNLEN      100
-/* MAX_PATHLEN = max length of a directory or file name */
-#define MAX_PATHLEN     300
+/* - Many limits can be changed here by adjusting the MAX_* constants      */
```

Comment:
This update was to improve readability

Reply:
Thanks, this looks much better now!

(I didn't know that you can have comments in the same line as the #define command)